modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,138 @@
+    namespace: openshift-cluster-csi-drivers
+  serviceAccountNames:
+  - aws-efs-csi-driver-operator
+  - aws-efs-csi-driver-controller-sa
+----
+////
+
+. Use the `ccoctl` tool to process all `CredentialsRequest` objects in the `credrequests` directory by running the following command:
++
+[source, terminal]
+----
+$ ccoctl aws create-iam-roles \
+  --name=<name> \
+  --region=<aws_region> \
+  --credentials-requests-dir=<path_to_directory_with_list_of_credentials_requests>/credrequests \ --identity-provider-arn=arn:aws:iam::<aws_account_id>:oidc-provider/<name>-oidc.s3.<aws_region>.amazonaws.com
+----
++
+where:
++
+--
+** `<name>` is the name used to tag any cloud resources that are created for tracking.
+** `<aws-region>` is the AWS region in which cloud resources will be created.
+** `<path_to_ccoctl_output_dir>` is the path to the public key file that the `ccoctl aws create-key-pair` command generated.
+** `<aws_account_id>` is the AWS account ID.
+-- 
++
+The command creates an IAM role in AWS for the Operator and a corresponding `<operator_credentialsrequest_name>-credentials.yaml` file in the `<path_to_ccoctl_output_dir>/manifests/` directory of the local file system.
+
+//This step is not in the ALB proc or the existing STS content. Needs SME assessment.
+. Create the Operator credentials and secret by running the following command:
++
+[source, terminal]
+----
+$ oc create -f <path_to_ccoctl_output_dir>/manifests/<operator_credentialsrequest_name>-credentials.yaml
+----
+
+. Apply the secrets to your cluster by running the following command:
++
+[source,terminal]
+----
+$ ls <path_to_ccoctl_output_dir>/manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+----
+
+.Verification
+
+//Not entirely sure this step makes sense in this version of the flow.
+* To verify that the Operator secrets are created, list the files in the `<path_to_ccoctl_output_dir>/manifests` directory:
++
+[source,terminal]
+----
+$ ll <path_to_ccoctl_output_dir>/manifests
+----
++
+.Example output:
++
+[source,terminal]
+----
+total 24
+-rw-------. 1 <user> <user> 161 Apr 13 11:42 cluster-authentication-02-config.yaml
+-rw-------. 1 <user> <user> 379 Apr 13 11:59 openshift-cloud-credential-operator-cloud-credential-operator-iam-ro-creds-credentials.yaml
+-rw-------. 1 <user> <user> 353 Apr 13 11:59 openshift-cluster-csi-drivers-ebs-cloud-credentials-credentials.yaml
+-rw-------. 1 <user> <user> 355 Apr 13 11:59 openshift-image-registry-installer-cloud-credentials-credentials.yaml
+-rw-------. 1 <user> <user> 339 Apr 13 11:59 openshift-ingress-operator-cloud-credentials-credentials.yaml
+-rw-------. 1 <user> <user> 337 Apr 13 11:59 openshift-machine-api-aws-cloud-credentials-credentials.yaml
+----
+//Not sure what this should show for the Operator use case.
+
+//This part should be right I think.
+You can verify that the IAM roles are created by querying AWS. For more information, refer to AWS documentation on listing IAM roles.
```

Comment:
The only part here that I feel confident makes sense in context is the last line (_You can verify that the IAM roles are created by querying AWS. For more information, refer to AWS documentation on listing IAM roles._). 

Does the `ll` of the `manifests` dir make sense in this version of the procedure? If it does, will need a new output showing an Operator example - this output is for cluster install.

Reply:
Ok, great - rearranged to accomplish this. Will just need some sample output for an `ll` in the Operator use case.